src/main_train.py

```diff
@@ -11,7 +11,8 @@
 
```

Comment:
As discussed in regards to #46:
- Add `CRITICAL` logger message when a required training column is missing.
- Update README with more information about training a model (such as which fields are required).
- Consider validating whether all required fields are present during processing of input file (to reduce waiting time before the error occurs).